player/lua/console.lua

```diff
@@ -576,6 +576,7 @@ local function render()
                   (font and '\\fn' .. font or '') ..
                   '\\fs' .. opts.font_size ..
                   '\\bord' .. opts.border_size .. '\\fsp0' ..
+                  '\\blur' .. (mp.get_property_native('osd-blur') * 720 / 288) ..
```

Comment:
stats also uses this ratio so that seems right. Maybe add a small comment explaining why.

Reply:
Done and also reordered some variables and added newlines.